lib/cocoapods/installer/file_references_installer.rb

```diff
@@ -153,7 +155,7 @@ def file_accessors
       #         The key of the group of the Pods project.
       #
       # @param  [Bool] reflect_file_system_structure_for_development
-      #         Wether organizing the a local pod's files in subgroups inside
+      #         Whether organizing a local pod's files in subgroups inside
```

Comment:
Ah, this would be nice. No this was actually kind of manually. RubyMine marks spelling mistakes but no grammar. 😓